src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,26 +51,125 @@ object ConductrPlugin extends AutoPlugin {
     )
 
   override def projectSettings: Seq[Setting[_]] =
-    super.projectSettings ++ List(
+    List(
       // Here we try to detect what binary universe we exist inside, so we can
       // accurately grab artifact revisions.
       isSbtBuild := Keys.sbtPlugin.?.value.getOrElse(false) && (Keys.baseDirectory in ThisProject).value.getName == "project",
 
+      discoveredDist <<= (dist in Bundle).storeAs(discoveredDist).triggeredBy(dist in Bundle),
+      discoveredConfigDist <<= (dist in BundleConfiguration).storeAs(discoveredConfigDist).triggeredBy(dist in BundleConfiguration)
+    )
+
+  override def buildSettings: Seq[Setting[_]] =
+    List(
       sandbox := sandboxTask.value.evaluated,
       sandboxRunTaskInternal := sandboxRunTask(ScopeFilter(inAnyProject, inAnyConfiguration)).value,
       conduct := conductTask.value.evaluated,
```

Comment:
By making these build tasks, won't the task be executed on all project aggregates? Just pointing this out as I remember noticing this in the Lagom plugin, and in fact we have code to prevent aggregation (see https://github.com/lagom/lagom/blob/master/dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala#L311-L317)